doc/rpmdeps.8

```diff
@@ -19,6 +19,35 @@ for the set of \fIFILE\fR arguments.
 Each \fIFILE\fR argument is searched for
 Elf32/Elf64, script interpreter, or per-script dependencies,
 and the dependencies are printed to stdout.
+.SH "OPTIONS"
+.TP
+\fB-P, --provides\fR
+Print the provides
+.TP
+\fB-R, --requires\fR
+Print the requires
+.TP
+\fB--recommends\fR
+Print the recommends
+.TP
+\fB--suggests\fR
+Print the suggests
+.TP
+\fB--supplements\fR
+Print the supplements
+.TP
+\fB--enhances\fR
+Print the enhances
+.TP
+\fB--conflicts\fR
+Print the conflicts
+.TP
+\fB--obsoletes\fR
+Print the obsoletes
+.TP
+\fB--alldeps\fR
+Print the alldeps
```

Comment:
There's no such thing as "the alldeps". 
--alldeps prints out all the dependencies in one go, using an entirely different formatting.